docs/using-kontena/volumes.md

```diff
@@ -26,6 +26,9 @@ See volume [scopes](#volume-scopes) for details on the different scopes.
 
 The driver has to be specified **without** a version tag, i.e. `rexray/s3fs:latest` is **not** supported. This is because using a hard version info on volume specification will make running multiple volume plugin versions within the grid impossible and also makes upgrading volume plugins really hard. Kontena will match the reported plugin version automatically and thus is able to create and manage the volumes.
 
+**NOTE:**
+There should be always only one version of a v2 volume plugin installed on a node at any given time. Having multiple versions installed will make volumes seen multiple times on a node and thus makes the automatic selection of correct volume and driver impossible. It is also advisable to install the plugins with `--alias` as that allows smoother (and less confusing) upgrades of the plugins.
```

Comment:
If not using the default `latest` version, then it is advisable to install the plugins with `--alias`, as that allows smoother (and less confusing) upgrades of the plugins.